pykickstart/commands/repo.py

```diff
@@ -423,5 +447,22 @@ def _getParser(self):
                     """ % versionToLongString(F27))
         return op
 
+class F30_Repo(F27_Repo):
+    removedKeywords = F27_Repo.removedKeywords
+    removedAttrs = F27_Repo.removedAttrs
+
+    def _getParser(self):
+        op = F27_Repo._getParser(self)
+        op.add_argument("--sslcacert", version=F30, help="""
+                        Path to the directory or file containing the
+                        certificate authorities to verify SSL certificates.""")
```

Comment:
It might be good to more clearly specify which file formats & directory layouts are supported for this.

Reply:
Hm good point. I took this documentation from `man dnf.conf`. It sadly doesn't specify further what is allowed.

Do you think it's enough to point to "what dnf accepts", or should we be more strict?

Reply:
I agree with Martin here, the pykickstart help string should explain more.  I do not think it is sufficient to say just look at the man page for dnf.conf because, historically, the yum and dnf documentation has also been lacking.  Plus it's kind of annoying when you're reading some docs and are told to go jump in to another doc for one specific thing.  It's ok to say "see also dnf.conf(5)" but I would prefer that the pykickstart documentation be explicit.  This information makes its way in to both the Fedora and RHEL install guides as well.